client/src/health/epics/pingEpic.ts

```diff
@@ -0,0 +1,42 @@
+import { Epic, ofType } from 'redux-observable';
+import { Observable, of } from 'rxjs';
+import { catchError, map, switchMap } from 'rxjs/operators';
+import GraphQLResponse from '../../shared/types/GraphQLResponse.type';
+import graphQLSubscriptionClient from '../../shared/utils/graphQLSubscriptionClient';
+import HealthActionType from '../actionTypes/Health.actionType';
+import HealthAction from '../actions/Health.action';
+import GraphQLPing from '../types/GraphQLPing.type';
+
+const subscribePing$ = (query: string): Observable<GraphQLResponse<GraphQLPing>> => {
+  return new Observable((observer) => {
+    graphQLSubscriptionClient.subscribe(
```

Comment:
The subscriptions client subscribe function returns a cleanup function which should be the return value of the observable observer setup (check out the Observable recipe in the lib readme). If you dont return this function, using `observable.unsubscribe()` wont work.
```suggestion
    return graphQLSubscriptionClient.subscribe(
```

Reply:
Thank you so much @enisdenjo , that is exactly what I want to add next step for unsubscribing! : )